src/H21.Wellness/ScavengerHuntItems.json

```diff
@@ -15,71 +15,84 @@
     "id": "6daed7d0-ee79-4ab9-98cc-70f48bab899d",
     "name": "Playground",
     "description": "Play ground",
-    "synonyms": [ "outdoor", " outdoor play equipment", "playground slide" ]
+    "synonyms": [ "outdoor", "outdoor play equipment", "playground slide" ]
   },
   {
     "id": "d036b012-5bc7-40ba-a7d7-91446b4585cd",
     "name": "Creek",
-    "description": "Creek"
+    "description": "Creek",
+    "synonyms": [ "water", "river", "lake", "bank", "nature", "trail", "stream" ]
```

Comment:
synonyms like nature wont work because then any picture of nature, like a tree, will trigger a success